features/flows/elastic/src/main/java/org/opennms/netmgt/flows/elastic/AggregateIndexInitializer.java

```diff
@@ -29,9 +29,12 @@
 
 import io.searchbox.client.JestClient;
 
+import java.nio.file.Paths;
+
 public class AggregateIndexInitializer extends DefaultTemplateInitializer {
 
-    public static final String TEMPLATE_RESOURCE = "/netflow_agg-template";
+    private static final String FLOW_TEMPLATE_DIR = System.getProperty("flow.templates.dir","");
```

Comment:
Please prefix the property:
```suggestion
    private static final String FLOW_TEMPLATE_DIR = System.getProperty("org.opennms.flow.templates.dir","");
```